spicy/runtime/src/base64.cc

```diff
@@ -35,7 +35,7 @@ hilti::rt::Bytes Stream::encode(const hilti::rt::Bytes& data) {
     if ( ! _state )
         throw Base64Error("encoding already finished");
 
-    std::string buf(data.size() * 2, {});
+    std::string buf(static_cast<std::size_t>(data.size() * 2), {});
     auto len = base64_encode_block(data.data(), static_cast<int>(data.size()), buf.data(), &_state->estate);
```

Comment:
Not changed by you, but could you add a patch here so we cast to the correct target type for `length_in`?

```suggestion
    auto len = base64_encode_block(data.data(), static_cast<size_t>(data.size()), buf.data(), &_state->estate);
```

There should be three additional instances of this.